domain/src/main/java/com/open/domain/Weather.kt

```diff
@@ -0,0 +1,32 @@
+package com.open.domain
+
+import com.google.gson.annotations.SerializedName
+
+
+data class Weather(
+    @SerializedName("weather") var weather: ArrayList<WeatherResponse> = arrayListOf(),
+    @SerializedName("main") var main: Main? = Main(),
+
+    )
+
+
+data class WeatherResponse(
+    @SerializedName("id") var id: Int? = null,
+    @SerializedName("main") var main: String? = null,
+    @SerializedName("description") var description: String? = null,
+    @SerializedName("icon") var icon: String? = null
+)
+
+
+data class Main(
+    @SerializedName("temp") var temp: Double? = null,
+    @SerializedName("feels_like") var feelsLike: Double? = null,
+    @SerializedName("temp_min") var tempMin: Double? = null,
+    @SerializedName("temp_max") var tempMax: Double? = null,
+    @SerializedName("pressure") var pressure: Int? = null,
+    @SerializedName("humidity") var humidity: Int? = null,
+    @SerializedName("sea_level") var seaLevel: Int? = null,
+    @SerializedName("grnd_level") var grndLevel: Int? = null
+
+)
```

Comment:
We don't need `Serialization` in the `domain` layer. It's whole job is to just describe the system's blueprint using pure Java/Kotlin code.